### DISCUSSION TITLE
I created prompts that required an excel sheet upload

### DISCUSSION THREAD

**Discussion by Haqbani** [danielmiessler/Fabric#596] [Q&A]
Hi all, I created a prompts that analyze an excel sheet. How can I direct fabric to the excel sheet?

Thanks!

**Comment by Haqbani** [comment]
any help ?

**Comment by Haqbani** [comment]
any idea ??

**Comment by AO12612** [comment]
One obvious answer would be to check if it makes sense to export to csv and use that text for fabric.
Maybe some more detail on what the prompt is trying to achieve?
